javascript/javascript-7-oop.md

```diff
@@ -321,8 +321,19 @@ The `Loan` constructor is using the `calculateDueDate()` method polymorphically.
 Instructor demonstrates in the video walkthrough how to work with a Class in JavaScript.
 
 ### Independent Practice
-Work through [this interactive lesson on Advanced JavaScript Objects](https://www.codecademy.com/courses/introduction-to-javascript/lessons/advanced-objects/exercises/adv-intro?action=resume_content_item).
-Work through [this Introduction to Classes on CodeAcademy](https://www.codecademy.com/courses/learn-javascript-classes/lessons/classes/exercises/introduction?course_redirect=introduction-to-javascript).
+* Work through [this interactive lesson on Advanced JavaScript Objects](https://www.codecademy.com/courses/introduction-to-javascript/lessons/advanced-objects/exercises/adv-intro?action=resume_content_item).
+
+* Work through [this Introduction to Classes on CodeAcademy](https://www.codecademy.com/courses/learn-javascript-
+classes/lessons/classes/exercises/introduction?course_redirect=introduction-to-javascript).
+
+* Find definition of following terms on your own:
+   * Namespace
+   * Hoisting
+   * Arity
```

Comment:
Just curious, are you trying to establish the fact that the arity property which was used to return the number of arguments expected by the function, no longer exists and has been replaced by the Function.prototype.length property?? @alodahl What do you think, what is good to mention here, arity or Function.prototype.length?

Reply:
I would say to replace arity by Function.prototype.length.

Reply:
Anyways as a concept/definition it is well and good to explore about arity. But you should establish the fact specific to javascript. You can suggest to go through the link: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Function/arity after mentioning Arity.

Reply:
Thank You for pointing out. Arity is now replaced by Function.prototype.length but Arity can also be used. I will add it too.